scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -846,18 +846,40 @@ export function AuthoringDirective(
                 $scope.autosave(item);
             };
 
+            Object.values(extensions).forEach((extension) => {
+                if (extension.activationResult?.contributions?.authoring != null) {
+                    $scope.onUpdateFromExtension = extension.activationResult.contributions.authoring.onUpdate;
```

Comment:
It should support defining `authoring.onUpdate` in multiple extensions. As you've implemented it now, it will only call `onUpdate` for one extension - the one which ends up being the last iteration in `Object.values(extensions).forEach`.

It should also pipe values through extensions so if `next.priority` is 3, but first extension returns `Promise.resolve({...item, priority: 5})`, the second extension should receive 5 as `priority` in its `next` argument.

See an example here https://github.com/superdesk/superdesk-client-core/pull/3262/files#diff-374a2e82b3cfbcea00b03816fc47b614R120